### PR TITLE
Admins country fix

### LINKED ISSUE #1903: admin info missing or incorrect in the Philippines and Malta
## Issue
When testing - noticed the `drive_on_right` flag was incorrect in the Philippines. When looking at the admin info - it was missing:

image

```
node info
lon | 120.99157
lat | 14.584733
```

Noticed that some routes in the Philippines and Malta will alternate the `drive_on_right` value.

## Fix the following:

- [ ] Philippines
- [ ] Malta

### PR DESCRIPTION
**Pull request by gknisely**
# Issue

Fixes drive on right vs drive on left issues.  

Fixes #1903 

## Tasklist

 - [ ] Add tests
 - [ ] Review - you must request approval to merge any PR to master
 - [x] Add #fixes with the issue number that this PR addresses
 - [ ] Generally use squash merge to rebase and clean comments before merging
 - [ ] Update the [changelog](CHANGELOG.md)
 - [ ] Update relevant [documentation](docs/README.md)

### CODE REVIEW

**Comment by kdiluca** [comment]
@gknisely I believe we need to update the lua for Ireland as well as it's still showing driving_side as right.  
image

**Comment by kdiluca** [comment]
@gknisely @dgearhart I tested Ireland with latest admin.sqlite as well and we are good to go.
image
image

**Comment by kdiluca** [comment]
Tests pass now:  https://paper.dropbox.com/doc/20190806_213111_production_test_routes_20190801_300_standardGuidance-vs.-20190806_213111_production_test_routes_20190801_300_sharedGuidance-driving-side-diffs--AjWehdedXZP2fgtIQ9eaoJ4lAQ-LX1xOeJOXfIrKkzg4sE2p

File: src/mjolnir/graphbuilder.cc [hunk 1]
```diff
@@ -982,6 +983,14 @@ void BuildTileSet(const std::string& ways_file,
 
         graphtile.nodes().back().set_timezone(tz_index);
 
+        // if you need to look at the attributes for nodes, grab the LL and update the if statement.
```
Comment (dgearhart) [inline src/mjolnir/graphbuilder.cc]: thanks

**Pull request merged**